apps/cowswap-frontend/src/modules/twap/const.ts

```diff
@@ -6,6 +6,8 @@ import ms from 'ms.macro'
 
 import { TwapOrderExecutionInfo, TwapOrderStatus } from './types'
 
+import { MAX_CUSTOM_DEADLINE } from '../limitOrders/pure/DeadlineSelector/deadlines'
```

Comment:
TWAP module shouldn't depend on Limit orders module.
I think better to do:
```
export const MAX_PART_TIME = 1000
```

Reply:
Good point, I missed that!
I'll move this up to a common const as it's needed on both modules.